pkgs/development/tools/marksman/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, fetchFromGitHub
+, buildDotnetModule
+, dotnetCorePackages
+, marksman
+, testVersion
+}:
+
+buildDotnetModule rec {
+  pname = "marksman";
+  version = "2022-10-30";
+
+  src = fetchFromGitHub {
+    owner = "artempyanykh";
+    repo = "marksman";
+    rev = version;
+    sha256 = "sha256-Ko/dHV0mxRYukr9vdQD7HSDOYdqfc/DcuU8XPCYkSWA=";
+  };
+
+  patches = [ ./patches/0001-revert-embed-git-commit-into-the-version.patch ];
+
+  projectFile = "Marksman/Marksman.fsproj";
+
+  doCheck = true;
+  testProjectFile = "Tests/Tests.fsproj";
+
+  nugetDeps = ./deps.nix;
+
+  dotnet-sdk = dotnetCorePackages.sdk_6_0;
+  dotnet-runtime = dotnetCorePackages.runtime_6_0;
+
+  postInstall = ''
+    install -m 644 -D -t $out/share/doc LICENSE
+  '';
+
```

Comment:
This will conflict with any other program that puts a file there and create needless conflicts.

```suggestion
```

Reply:
How about

```
postInstall = ''
    install -m 644 -Dt $out/share/doc/marksman/ LICENSE
'';
```

instead then? At least on my system a few packages seem to already be placing license files along with documentation there, and this matches what other distros do.

Reply:
Yes, if we want to put a LICENSE, it'll have to be `$out/share/doc/marksman`. I didn't notice that it's in common dir, my bad.